web/js/map/data/results.js

```diff
@@ -671,20 +680,20 @@ export function dataResultsTagURS(spec) {
     }
     granule.urs = isURS;
     if (isURS) {
-      meta.urs = (meta.urs) ? meta.urs += 1 : 1;
+      meta.urs = meta.urs ? (meta.urs += 1) : 1;
     }
     return granule;
   };
 
   return self;
-};
+}
 
 export function dataResultsTagVersion() {
   var self = {};
 
   self.name = 'TagVersion';
 
-  self.process = function (meta, granule) {
```

Comment:
Did a linting rule change for us to be able to use this style syntax, i.e. no space? Personally I like this style but I thought it was set in the semistandard. Not advocating a change here, just curious.

Reply:
Not sure. Maybe it was just a warning and that page hadn't been edited in a while?